src/Microsoft.OData.CodeGen/Templates/ODataT4CodeGenerator.cs

```diff
@@ -1992,7 +1992,7 @@ internal void WriteNamespace(string fullNamespace)
                 {
                     IEdmTypeReference edmTypeReference = function.Parameters.First().Type;
 
-                    if (this.context.ExcludedBoundOperations.Contains($"{function.Name}({edmTypeReference.Definition.FullTypeName()})"))
+                    if (this.context.ExcludedBoundOperations != null && this.context.ExcludedBoundOperations.Contains($"{function.Name}({edmTypeReference.Definition.FullTypeName()})"))
```

Comment:
Alternatively,
```suggestion
                    if (this.context.ExcludedBoundOperations?.Contains($"{function.Name}({edmTypeReference.Definition.FullTypeName()})") == true)
```
Applies to other similar changed lines in _src/Microsoft.OData.CodeGen/Templates/ODataT4CodeGenerator.cs_ and _src/Microsoft.OData.CodeGen/Templates/ODataT4CodeGenerator.ttinclude_?